js/views/moderatorCard.js

```diff
@@ -7,8 +7,12 @@ import { launchModeratorDetailsModal } from '../utils/modalManager';
 
 export default class extends BaseVw {
   constructor(options = {}) {
-    super(options);
-    this.options = options;
+    const opts = {
```

Comment:
Did you have some default options and then removed them here?

As it is now, this assignment is pointless since it results in the same thing, so you could have left the code as it was before.